### PR TITLE
Add node-controller to TPR

### PR DESCRIPTION
**Pull request by r7vme**
Root issue: https://github.com/giantswarm/k8scloudconfig/issues/162

All PRs (all tested together):
https://github.com/giantswarm/kvm-operator-node-controller/pull/4
https://github.com/giantswarm/kubernetesd/pull/289
https://github.com/giantswarm/kvm-operator/pull/221
https://github.com/giantswarm/kvmtpr/pull/51

### CODE REVIEW

File: spec/kvm.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import "github.com/giantswarm/kvmtpr/spec/kvm"
 type KVM struct {
 	EndpointUpdater kvm.EndpointUpdater `json:"endpointUpdater" yaml:"endpointUpdater"`
 	K8sKVM          kvm.K8sKVM          `json:"k8sKVM" yaml:"k8sKVM"`
+	NodeController  kvm.NodeController  `json:"nodeController" yaml:"nodeController"`
```
Comment (xh3b4sd) [inline spec/kvm.go]: I am so bad at this, I had to look it up because I was not sure. The internet says N comes after M in the alphabet. :D

Comment (r7vme) [reply spec/kvm.go]: What? E - K - N all good, isn't it?

Comment (xh3b4sd) [reply spec/kvm.go]: But it is M - N and not N - M. :D

Comment (r7vme) [reply spec/kvm.go]: hehe, i opened whole file now. I see it i see it, i was blind.

;)

**Comment by r7vme** [comment]
@xh3b4sd done.

File: spec_test.go [hunk 2]
```diff
@@ -135,6 +136,11 @@ func TestSpecYamlEncoding(t *testing.T) {
 				},
 				"hostPath",
 			},
+			NodeController: kvm.NodeController{
+				nodecontroller.Docker{
+					Image: "quay.io/giantswarm/kvm-operator-node-controller",
+				},
+			},
```
Comment (xh3b4sd) [inline spec_test.go]: Here and below please as well.

**Comment by r7vme** [comment]
Done

**Pull request merged**